plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * OrdersTableDataStore class file.
+ */
+
+namespace Automattic\WooCommerce\Internal\DataStores\Orders;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class is the standard data store to be used when the custom orders table is in use.
+ */
+class OrdersTableDataStore extends \Abstract_WC_Order_Data_Store_CPT implements \WC_Object_Data_Store_Interface, \WC_Order_Data_Store_Interface {
```

Comment:
Why subclass `Abstract_WC_Order_Data_Store_CPT`? Are we planning to use the metadata methods for the EAV fallback table?

Reply:
I just subclassed `Abstract_WC_Order_Data_Store_CPT` since that's what the existing data store class was doing, but I was not sure if that was actually the correct thing to do. What would this class extend/implement instead?

Reply:
Without a specific use of the CPT abstract in mind it's probably safer not to subclass it (so we don't accidentally use the old CPT methods). I think the interfaces should remain.